src/test/java/org/folio/support/base/IntegrationTestBase.java

```diff
@@ -60,15 +63,19 @@ public class IntegrationTestBase {
   protected static OkapiConfiguration okapi;
   protected static KafkaTemplate<String, String> kafkaTemplate;
   protected static ObjectMapper objectMapper;
+  protected static DatabaseHelper databaseHelper;
+  protected static final String TENANT = "test";
```

Comment:
Constant for tenant already exists: org.folio.support.base.TestConstants.TENANT_ID